CMake/SlicerBlockPlatformCheck.cmake

```diff
@@ -34,42 +34,26 @@ endif()
 
 if(Slicer_PLATFORM_CHECK)
   # See CMake/Modules/Platform/Darwin.cmake)
-  #   6.x == Mac OSX 10.2 (Jaguar)
-  #   7.x == Mac OSX 10.3 (Panther)
-  #   8.x == Mac OSX 10.4 (Tiger)
-  #   9.x == Mac OSX 10.5 (Leopard)
-  #  10.x == Mac OSX 10.6 (Snow Leopard)
-  #  11.x == Mac OSX 10.7 (Lion)
-  #  12.x == Mac OSX 10.8 (Mountain Lion)
-  if(DARWIN_MAJOR_VERSION LESS "9")
-    message(FATAL_ERROR "Only Mac OSX >= 10.5 is supported !")
+  # and https://en.wikipedia.org/wiki/Darwin_(operating_system)#Release_history
+  #  17.x == macOS 10.13 (High Sierra)
+  #  18.x == macOS 10.14 (Mojave)
+  #  19.x == macOS 10.15 (Catalina)
+  #  20.x == macOS 11 (Big Sur)
+  #  21.x == macOS 12 (Monterey)
+  if(DARWIN_MAJOR_VERSION LESS "17")
+    message(FATAL_ERROR "Only macOS >= 10.13 is supported !")
   endif()
 
   if(MSVC)
-    # See http://www.cmake.org/cmake/help/v2.8.10/cmake.html#variable:MSVC_VERSION
+    # See https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html
     # and https://en.wikipedia.org/wiki/Microsoft_Visual_Studio#Version_history
-    #   1200 = VS  6.0 (Visual Studio 6.0)
-    #   1300 = VS  7.0 (Visual Studio .NET (2002))
-    #   1310 = VS  7.1 (Visual Studio .NET 2003)
-    #   1400 = VS  8.0 (Visual Studio 2005)
-    #   1500 = VS  9.0 (Visual Studio 2008)
-    #   1600 = VS 10.0 (Visual Studio 2010)
-    #   1700 = VS 11.0 (Visual Studio 2012)
-    if(MSVC_VERSION VERSION_LESS 1500)
-      message(FATAL_ERROR "Visual Studio >= 2008 is required !")
+    #   1900 = VS 14.0 (Visual Studio 2015)
+    #   1910-1919 = VS 15.0 (Visual Studio 2017)
+    #   1920-1929 = VS 16.0 (Visual Studio 2019)
+    #   1930-1939 = VS 17.0 (Visual Studio 2022)
+    if(MSVC_VERSION VERSION_LESS 1900)
```

Comment:
```suggestion
  if(NOT MSVC_VERSION VERSION_GREATER_EQUAL 1900)
```

Reply:
I have update the syntax based on this recommendation